app/services/finance/stripe_charge_service.rb

```diff
@@ -47,7 +52,7 @@ def create_payment_intent(amount)
 
   def confirm_payment_intent(payment_intent)
     # Passing the gateway option `off_session: false` will cause a `requires_action` status on the payment intent in cases where otherwise it would be `requires_payment_method`.
-    # This happens even when the payment intent has been originally created with `off_session: true` – i.e. Stripe allows us to turn an "off_session" payment intent into an "on_session" one.
+    # This happens even when the payment intent has been originally created with `off_session: true` - i.e. Stripe allows us to turn an "off_session" payment intent into an "on_session" one.
```

Comment:
Do we need such minor changes? I prefer us not changing lines like this as it makes following `git blame` harder.

Reply:
Must be an auto-fix thing... right? 😬